consumers/src/unit-test/java/uk/gov/moj/sdt/cmc/consumers/CMCConsumerGatewayTest.java

```diff
@@ -80,24 +84,34 @@ void shouldInvokeBreathingSpace() throws Exception {
     @Test
     void shouldInvokeJudgementRequest() throws Exception {
         JudgementResponse response = new JudgementResponse();
-        response.setJudgmentEnteredDate(Calendar.getInstance().getTime());
-        response.setFirstPaymentDate(Calendar.getInstance().getTime());
+        Date date = formattedDate();
+        response.setJudgmentEnteredDate(date);
+        response.setFirstPaymentDate(date);
         when(judgementService.requestJudgment(anyString(), any(), any())).thenReturn(response);
 
         IIndividualRequest individualRequest = mock(IIndividualRequest.class);
         JudgementRequest judgementRequest = mock(JudgementRequest.class);
         when(individualRequest.getRequestPayload()).thenReturn(XML);
         when(individualRequest.getSdtRequestReference()).thenReturn("MCOL-0000001");
         when(xmlToObject.convertXmlToObject(anyString(), any())).thenReturn(judgementRequest);
+        when(xmlToObject.convertObjectToXml(response)).thenReturn("XML");
         when(individualRequest.getRequestType()).thenReturn(RequestType.JUDGMENT.getType());
 
         cmcConsumerGateway.individualRequest(individualRequest, CONNECTION_TIME_OUT, RECEIVE_TIME_OUT);
 
         verify(judgementService).requestJudgment(anyString(), any(), any(JudgementRequest.class));
         verify(xmlToObject).convertXmlToObject(anyString(), any());
+        verify(xmlToObject).convertObjectToXml(any(JudgementResponse.class));
         verify(individualRequest).getRequestPayload();
         verify(individualRequest).getSdtRequestReference();
         verify(individualRequest).getRequestType();
+        verify(individualRequest).setTargetApplicationResponse(anyString());
```

Comment:
The setTargetApplicationResponse() method should be called with the result of convertObjectToXml(), so anyString() can be replaced with "XML".

Reply:
addressed